app/cdap/text/text-en.yaml

```diff
@@ -3432,8 +3432,41 @@ features:
       success: You have successfully created the pipeline "{pipelineName}".
     Skip: Skipped
   WranglerNewUI:
+    FooterPanel:
+      labels:
+        message: Current data - {rowCount} rows and {columnCount} columns
+        columnViewPanel: Column View Panel
+        directives: Directives
+        recipeSteps: Recipe Steps
+        zoomPercent100: 100%
     NoRecordScreen:
       gridTable:
         title: No rows left in this sample
         subtitle: Remove some recipe steps to view some data
+    GridPage:
+      directivePanel:
+        moreInfoOnDirective: More info to this directive
+        usage: Usage
+        inputDirective: Input a directive
+    Snackbar:
+      labels:
+        datasetSuccess: Dataset loaded successfully 
+        failure: Failure
+        retrieveFailure: Failed to retrieve sample data for
+        sampleFailure: Failed to retrieve sample data
+        success: Success
+        undo: Undo
+        workspaceFailure: Failed to create workspace for
+  WranglerNewRecipeSteps:
+    labels:
+       recipe: Recipe
+       serialNo: S.No.
```

Comment:
This should not be abbreviated; could just say `Step` or even `#`

Reply:
Since the other header says `Recipe steps`, maybe we should review both

Reply:
Replaced S.No. with # , we can review it if any changes required